components/eventing-controller/testing/subscriber.go

```diff
@@ -62,7 +73,7 @@ func (s *Subscriber) Shutdown() {
 	go func() {
 		err := s.server.Close()
 		if err != nil {
-			s.t.Errorf("failed to shutdown Subscriber: %v", err)
+			log.Printf("failed to shutdown Subscriber: %v", err)
```

Comment:
If we `log.Fatal()` it would fail the test? If yes, I think it would be better to fail the test.

Reply:
Shutdown of the sever is more of a nice-to-have IMO. If shutdown fails, the test is still good hence I'm against `log.Fatal`